app/src/main/java/xyz/jhughes/laundry/LaundryParser/Machine.java

```diff
@@ -21,7 +23,10 @@ public Machine(String name, String type, String status, String time) {
     }
 
     public String getName() {
-        return name;
+        // Get rid of leading 0s
+        if (name.matches(".*0[1-9][0-9].*")) return name.replaceFirst("0", "");
+        else if (name.matches(".*0{2}[1-9].*")) return name.replaceAll("0", "");
```

Comment:
I could. But I agree, a server change would be better, and would eliminate the above concern. But isn't the server Tylers, not ours? Or did he give the code/ownership to us?